openedx_certificates/queue_xqueue.py

```diff
@@ -55,6 +55,7 @@ def get_length(self):
                 self.url), params={'queue_name': self.queue_name})
             response = json.loads(request.text)
             if response['return_code'] != 0:
+                log.info('Response: {0}'.format(request.text))
```

Comment:
@ChrisChV you mention this is the response, but you are showing the `request.text`

Reply:
OMG! Sorry about that @eLRuLL I made the change :+1: